var/spack/repos/builtin/packages/nimrod-aai/package.py

```diff
@@ -12,17 +12,16 @@ class NimrodAai(CMakePackage):
     enabled by modern Fortran.
     """
 
-    homepage = "https://gitlab.com/NIMRODteam/nimrod-abstract"
-    url = (
-        "https://gitlab.com/NIMRODteam/nimrod-abstract/-/archive/23.9/nimrod-abstract-23.9.tar.gz"
-    )
-    git = "https://gitlab.com/NIMRODteam/nimrod-abstract.git"
+    homepage = "https://gitlab.com/NIMRODteam/open/nimrod-abstract"
+    url = "https://gitlab.com/NIMRODteam/open/nimrod-abstract/-/archive/24.2/nimrod-abstract-24.2.tar.gz"
+    git = "https://gitlab.com/NIMRODteam/open/nimrod-abstract.git"
 
     maintainers("jacobrking")
 
     version("main", branch="main")
-    version("23.9", sha256="212d591c5a5e7a394b56a5cf2f92cc69feafc49dd5f042fa95eeb6441649390b")
-    version("23.6", sha256="1794b89a5a64ff2b3c548818b90d17eef85d819ba4f63a76c41a682d5b76c14f")
+    version("24.2", sha256="1dd4d51426f141c058e25cb29870eaf15e0edfb44d80df94e7c65c850ca78eda")
+    version("23.9", sha256="34f7ee00bbbe9a6d08304473e8893af9bd94af8dbd0bbd50b8b441057023e179")
+    version("23.6", sha256="de7e5c5cc2ad97dc0e66628d29c8153fa807821a316eb9aa8ee21a39c69df800")
```

Comment:
Why the shasum of old version changed?

Reply:
I'm not sure. The package files didn't. The only reason that I can think of is that is it associated with the moving the repo into a subgroup.

Reply:
The older 23.6 tar.gz file from the mirror (https://mirror.spack.io/_source-cache/archive/17/1794b89a5a64ff2b3c548818b90d17eef85d819ba4f63a76c41a682d5b76c14f.tar.gz) extracts into a subdirectory e.g. `nimrod-abstract-23.6/`. After the move and rename the tar.gz file extracts into `nimrod-aai-23.6-e0fac728c5eba2b08ff564e95776241376ac85ac/`, but there are no other changes inside that directory based on a recursive diff.

Because the older 23.9 wasn't on mirror.spack.io (https://mirror.spack.io/_source-cache/archive/21/212d591c5a5e7a394b56a5cf2f92cc69feafc49dd5f042fa95eeb6441649390b.tar.gz), I wasn't able to verify this for version 23.9, and I have to assume that the same applies for 23.9.

Based on this assessment and since there are no other outstanding issues here I'll approve and leave to @alalazo to merge.